package.json

```diff
@@ -32,16 +31,19 @@
     "access": "public"
   },
   "dependencies": {
-    "@asyncapi/generator-filters": "^2.1.0",
-    "@asyncapi/generator-react-sdk": "^0.2.23",
-    "@asyncapi/parser": "^2.1.0",
+    "@asyncapi/generator-react-sdk": "0.2.25",
+    "@asyncapi/parser": "^2.1.0-next-major-spec.6",
     "openapi-sampler": "^1.3.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "yaml": "^1.10.2"
   },
   "devDependencies": {
-    "@asyncapi/generator": "^1.10.12",
+    "@asyncapi/generator": "https://github.com/asyncapi/generator#add-v3-support",
```

Comment:
This is needed to run the local `npm run start` command. See https://github.com/asyncapi/generator/pull/1017